scripts/linters/pre_commit_linter.py

```diff
@@ -420,12 +421,52 @@ def categorize_files(file_paths):
 
 
 def _print_complete_summary_of_lint_messages(lint_messages):
-    """Print complete summary of lint messages."""
+    """Print summary of lint messages."""
+    excluded_phrases = ['SUCCESS', 'FAILED', '[htmllint]', 'Linting CSS files.',
+                        'There are no JavaScript or Typescript files to lint.',
```

Comment:
Why do we add these logs in the lint_message if we are about to exculde it?

Reply:
We need these to check if all the tests are passing or not.

Reply:
I still get your answer @Hudda -- shouldn't we remove these from lint messages rather than removing it from here?

Reply:
> We need these to check if all the tests are passing or not.

In that case why not a flag (or boolean value) instead of relying on a string and removing it later?

Reply:
SUCCESS and FAILURE messages are still going to be printed. Just not under the 'Fix below errors' section, as Sean finds it inappropriate to have some SUCCESS messages under this section as this section only prints error messages. So that users do not have to scroll up frequently.

Reply:
I think we had a discussion on this i.e, we planned on scope the message trimming on linter classes. I don't see any such changes. Can you please explain what's the status on that?

Reply:
@DubeySandeep If we move the trimmer function to the respective files, then we are having the following problems:
1. We still going to need the trimming method in the main linter file(pre_commit_linter.py) to remove parts such as `started`, `ended`, and timestamps.
2. Moving method to respective files leads to the printing of linter messages twice, once trimmed and once without trimming. So it is not feasible to use this method there.
I think it is best to use this method here.

Reply:
> We still going to need the trimming method in the main linter file(pre_commit_linter.py) to remove parts such as started, ended, and timestamps.

yeah, that's what I'm asking for i.e, each linter will filter out their own message, so in this case, pre_commit_liter will filter out its own message (in the case where we want it to filter the message).

> Moving method to respective files leads to the printing of linter messages twice, once trimmed and once without trimming. So it is not feasible to use this method there.

Any specific reason for this? Have you found why this is happening?

Reply:
@DubeySandeep This is somehow related to multithreading. concurrent_utils_task is appending messages in the PRINT() function to the linter_messages list.